templates/common/.github/workflows/java/terraform/azure-dev.yml

```diff
@@ -28,6 +28,24 @@ jobs:
       - run: |
           chmod +x ${GITHUB_WORKSPACE}/src/api/mvnw
 
+      - name: Install Nodejs
+        uses: actions/setup-node@v3
+        with:
+          node-version: 16
+
+      - name: Login az
+        uses: azure/login@v1
+        with:
+          creds: ${{ secrets.AZURE_CREDENTIALS }}
+
+      - name: Set az account
+        uses: azure/CLI@v1
+        with:
+          inlineScript: |
+            az account show
```

Comment:
Do we need to run `account show` before and after?

Reply:
Will update. 
That was part of me testing it :)